Backend/dev/migrations/dynamic-offer-driver-app/0437-add-notification-config-values.sql

```diff
@@ -18,11 +18,11 @@ VALUES
 (atlas_driver_offer_bpp.uuid_generate_v4(), '7f7896dd-787e-4a0b-8675-e9e6fe93bb8f', '1e7b7ab9-3b9b-4d3e-a47c-11e7d2a9ff98', 'ENGLISH', 'RIDE_SCHEDULED', 'Z9', '{#isRentalOrIntercity#} Ride Allocated', 'Reach Customer pickup location before 15 minutes from scheduled ride start time\n Be online before 45 minutes of Scheduled Ride Start\n Cancellation before 1 hr from the scheduled ride start will lead to penalties on the platform.\n Sakkath Service ! Namma Duty !', 'Okay'),
 (atlas_driver_offer_bpp.uuid_generate_v4(), '7f7896dd-787e-4a0b-8675-e9e6fe93bb8f', '1e7b7ab9-3b9b-4d3e-a47c-11e7d2a9ff98', 'ENGLISH', 'RIDE_SCHEDULED_REMINDER', 'Z9', 'Scheduled {#isRentalOrIntercity#} Ride Allocated', 'Reach Customer pickup location before 15 minutes of ride start time\n Be online before 45 minutes of Scheduled Ride Start\n Cancellation before 1 hr from the scheduled ride start will lead to penalties on the platform.\n Sakkath Service ! Namma Duty !', 'Okay');
 
-INSERT INTO atlas_driver_offer_bpp.merchant_push_notification (key, merchant_id, merchant_operating_city_id, title, body, language, fcm_notification_type)
+INSERT INTO atlas_driver_offer_bpp.merchant_push_notification (id, key, merchant_id, merchant_operating_city_id, title, body, language, fcm_notification_type)
 VALUES
```

Comment:
We can avoid manually changing existing queries. Should be handled with default condition while adding `id`.
```
ALTER TABLE atlas_driver_offer_bpp.merchant_push_notification ADD COLUMN id UUID DEFAULT md5(random()::text || clock_timestamp()::text)::uuid;
```